### PR TITLE
Add TR-FC1-ULAKBIM to vo.usegalaxy.eu

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

@hbayindir I am adding here the info about `vo.usegalaxy.eu` at `TR-FC1-ULAKBIM`.

I initially requested the mapping of this entitlement:

```yaml
urn:mace:egi.eu:group:vo.usegalaxy.eu:role=member#aai.egi.eu
```

But instead I realised it should be this one:
https://github.com/EGI-Federation/fedcloud-catchall-operations/blob/b852bd6ced9f03d6fbfda4a637753c4d34400cba/vo-mappings.yaml#L91

Could you please update it?

Many thanks!
Sebastian

---

**Related issue :**

### CODE REVIEW

**Comment by hbayindir** [comment]
@sebastian-luna-valero, the new entitlement is in place. You can continue as desired.

Cheers,

Hakan

**Comment by sebastian-luna-valero** [comment]
Thank you very much @hbayindir 

Let's then wait for an approval from the EGI team to get this merged.

**Review by enolfc — APPROVED** [review]
LGTM

**Pull request merged**
